### PR TITLE
EC2 Launch Configuration boto3 return values documentation

### PR DESCRIPTION
**Pull request by wilvk**
##### SUMMARY
As requested by @s-hertel , I have documented the return values for the module ec2_lc

##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
lib/ansible/modules/cloud/amazon/ec2_lc.py

##### ANSIBLE VERSION

```
ansible 2.4.0 (ec2_lc_boto3_ret_vals bca583adef) last updated 2017/07/19 22:01:29 (GMT +1100)
  config file = None
  configured module search path = [u'/Users/willvk/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/willvk/Source/gits/ansibles/ansible/lib/ansible
  executable location = /Users/willvk/Source/gits/ansibles/ansible/bin/ansible
  python version = 2.7.10 (default, Feb  7 2017, 00:08:15) [GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.34)]
```

### CODE REVIEW

**Comment by samdoran** [comment]
cc @s-hertel

File: lib/ansible/modules/cloud/amazon/ec2_lc.py [hunk 1]
```diff
@@ -151,26 +178,75 @@
...
+    type: str
+name:
+    description: The name of the launch configuration.
+    returned: Always
+    type: str
+result:
+    description: The resultant object describing the launch config returned from boto3.
+    returned: Always
+    type: str
+
```
Comment (s-hertel) [inline lib/ansible/modules/cloud/amazon/ec2_lc.py]: and security_groups, which is a list

File: lib/ansible/modules/cloud/amazon/ec2_lc.py [hunk 2]
```diff
@@ -151,26 +178,75 @@
...
+    returned: Always
+    type: str
+name:
+    description: The name of the launch configuration.
+    returned: Always
+    type: str
+result:
+    description: The resultant object describing the launch config returned from boto3.
+    returned: Always
+    type: str
```
Comment (s-hertel) [inline lib/ansible/modules/cloud/amazon/ec2_lc.py]: type: dict

**Review by s-hertel — CHANGES_REQUESTED** [review]
Just reviewing the documentation. This is a follow-up PR to 26348 which has the rest of the review.

File: lib/ansible/modules/cloud/amazon/ec2_lc.py (hunk 2 above)
Comment (wilvk) [reply lib/ansible/modules/cloud/amazon/ec2_lc.py]: fixed

File: lib/ansible/modules/cloud/amazon/ec2_lc.py (hunk 1 above)
Comment (wilvk) [reply lib/ansible/modules/cloud/amazon/ec2_lc.py]: added

**Review by s-hertel — APPROVED** [review]

**Comment by wilvk** [comment]
@s-hertel can this PR be closed now?

**Pull request closed without merging**